t/unit/backends/test_elasticsearch.py

```diff
@@ -80,3 +80,33 @@ def test_backend_params_by_url(self):
             assert x.scheme == 'elasticsearch'
             assert x.host == 'localhost'
             assert x.port == 9200
+
+    def test_index(self):
+        x = ElasticsearchBackend(app=self.app)
+        x.doc_type = 'test-doc-type'
+        x._server = Mock()
+        x._server.index = Mock()
+        expected_result = dict(
+            _id=sentinel.task_id,
+            _source={'result': sentinel.result}
+        )
+        x._server.index.return_value = expected_result
+
+        body = {"field1": "value1"}
+        x._index(id=sentinel.task_id, body=body, kwarg1='test1')
+        x._server.index.assert_called_once_with(
+            id=sentinel.task_id,
+            doc_type=x.doc_type,
+            index=x.index,
+            body=body,
+            kwarg1='test1'
+        )
+
+    def test_config_params(self):
+        self.app.conf.elasticsearch_max_retries = 10
+        self.app.conf.elasticsearch_timeout = 20.0
+        self.app.conf.elasticsearch_retry_on_timeout = True
+
```

Comment:
Since you are updating the configuration and these parameters are set in the constructor, you need to re-initialize the backend instance, so that it re-reads the configuration:

```
self.backend = ElasticsearchBackend(app=self.app)
```

This should resolve the build job errors.